src/main/java/com/myhome/bootstrap/DataLoader.java

```diff
@@ -59,7 +59,7 @@
 
   @Override public void run(String... args) throws Exception {
     //loadPostmanData();
-    loadData();
+    //loadData();
```

Comment:
Strange thing here if uncomment lines and run all test in IDEA, then lots of them fails, but build here is ok